components/molecule/dropdownOption/src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import AtomInput from '@s-ui/react-atom-input'
+
+const CLASS = 'sui-MoleculeDropdownOption'
+
+class MoleculeDropdownOption extends Component {
+  state = {
+    selected: this.props.selected
```

Comment:
I would change the name of the prop from `selected` to `initialSelected` so it clearly states that the value of the initialSelected is what is going to be used to initialize the state but future updates are not going to update the component.

Reply:
I agree with @miduga . If not, what about `defaultSelected` or `defaultOption`?